### DISCUSSION TITLE
Miden Rust Book

### DISCUSSION THREAD

**Discussion by gobidash** [0xPolygonMiden/compiler#115] [Ideas]
A long term idea but this is about creating something like the [Cairo book ](https://book.cairo-lang.org/)or the [Move book](https://move-language.github.io/move/)

**Comment by bobbinth** [comment]
Since the current plan is to compile from Rust, I'm not sure we need such broad docs. I think for us, Miden SDK docs (or something similar) would probably make sense. This would basically explain how to use Rust to code up smart contracts for Miden.
